tools/terraform/fleetdm_client/fleetdm_client.go

```diff
@@ -0,0 +1,276 @@
+package fleetdm_client
+
+// This file gives us a nice API to use to call FleetDM's API. It's focused
+// only on teams.
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+	"time"
+)
+
+var prefix = "/api/v1/fleet"
+var teamPrefix = prefix + "/teams"
```

Comment:
Those should probably be `const`, I don't think they're meant to be modified?

Reply:
Yep, fixed.

Reply:
You're right; changed.